frontend/public/components/configmap-and-secret-data.jsx

```diff
@@ -28,15 +29,25 @@ export class SecretData extends React.PureComponent {
     this.setState({ showSecret: !this.state.showSecret });
   }
 
+  getValue(rawValue) {
+    if (_.isNil(rawValue)) {
+      return <em className="text-muted">No value</em>;
```

Comment:
We shouldn't use italics if using text-muted

```jsx
return <span className="text-muted">No value</span>;
```

Reply:
:+1: haven't realized that we use it on non-italics